tests/test_misc.py

```diff
@@ -0,0 +1,64 @@
+"""
+Misc tests for helper functions.
+"""
+
+from __future__ import unicode_literals
+from __future__ import print_function
+
+import os
+from shutil import rmtree
+from tempfile import mkdtemp
+from unittest import TestCase
+
+from xbundle import exists, mkdir
+
+
+class TestXBundle(TestCase):
+    """
+    Misc tests for helper functions.
+    """
+
+    def test_exists(self):
```

Comment:
That's weird, I thought there was another function that was being used for that purpose but I can't find it now. I'll remove the test